src/_components/Assets/AddAssetContainer.jsx

```diff
@@ -24,6 +26,17 @@ import {
   filterModelNumbers
 } from '../../_utils/filterDropdowns';
 
+
+const initialFormState = {
```

Comment:
What is this for? I believe there's a better way of doing this, as opposed to having it here

Reply:
To clear state each time the component mounts. If state is not cleared as I explained in the PR then we end up with another bug. Another option is to use a lifecycle method I suppose? @sachungo

Reply:
Hmm...I still don't get it. Maybe you should look for me later to explain what is happening